core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -264,15 +300,18 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
                 </div>
               </div>
             </div>
-            <button class="btn open-in-editor-btn"
-                    ng-click="showExplorationEditor(exploration.id); $event.stopPropagation()">
-              Open In Editor
-            </button>
+            <a ng-href="/create/<[exploration.id]>">
```

Comment:
Put this href as a variable on scope instead of hard-coding it each time in the html ?